controllers/automation/policyautomation_controller.go

```diff
@@ -309,13 +309,13 @@ func (r *PolicyAutomationReconciler) Reconcile(
 			"clusterCount", strconv.Itoa(len(targetList)))
 
 		violationContext, _ := r.getViolationContext(ctx, policy, targetList, policyAutomation)
-
```

Comment:
Could you please not change the whitespace in unrelated parts of the code? If you want to commit this, then please add it in a separate commit.

Reply:
This comment also applies to elsewhere in the diff.